src/test/java/com/redhat/cloud/notifications/TestLifecycleManager.java

```diff
@@ -24,9 +24,11 @@
 
 public class TestLifecycleManager implements QuarkusTestResourceLifecycleManager {
 
+    // Keep the version synced with pom.xml.
+    private static final DockerImageName MOCK_SERVER_DOCKER_IMAGE = DockerImageName.parse("jamesdbloom/mockserver").withTag("mockserver-5.11.2");
```

Comment:
FYI `jamesdbloom/mockserver` is not new. We were already using that image before but with a much older version.